### PR TITLE
feat: bump CLIAPIVersion to 10

### PR DESCRIPTION
**Pull request by FrankYang0529**
#### Which issue(s) this PR fixes:

https://github.com/longhorn/longhorn/issues/7555

### CODE REVIEW

**Comment by FrankYang0529** [comment]
No need to update version to 10, because the version 9 has not released yet.

ref: https://github.com/longhorn/longhorn-engine/commit/01300d06c66365d833222cae7f05a03f1349e7d4#diff-41f3352cafc153a83763338fd3a03b513a9d04eb1ad077395c66ce96ca057e20

**Comment by FrankYang0529** [comment]
We still need the change, because we may backport the change to v1.5.x.

https://github.com/longhorn/longhorn-engine/blob/v1.5.x/pkg/meta/version.go

**Review by innobead — APPROVED** [review]

**Review by innobead — APPROVED** [review]

File: Dockerfile.dapper [hunk 1]
```diff
@@ -140,8 +140,7 @@ RUN cd /go/src/github.com/longhorn && \
     cd longhorn-instance-manager && \
     go build -o ./longhorn-instance-manager && \
     cp -r integration/rpc/ ${DAPPER_SOURCE}/integration/rpc/ && \
-    cp longhorn-instance-manager /usr/local/bin && \
-    cd ../ && rm -rf longhorn-instance-manager
+    cp longhorn-instance-manager /usr/local/bin
```
Comment (innobead) [inline Dockerfile.dapper:143]:
```suggestion
    install longhorn-instance-manager /usr/local/bin
```

**Review by innobead — APPROVED** [review]

**Review by innobead — APPROVED** [review]

**Comment by innobead** [comment]
cc @chriscchien need to update the upgrade test image.

@longhorn/qa

**Pull request merged**